kbase-extension/static/kbase/js/widgets/appWidgets2/paramResolver.js

```diff
@@ -1,6 +1,111 @@
-define(['bluebird', 'require'], (Promise, require) => {
+define([
+    'bluebird',
+    'require',
+    './input/checkboxInput',
```

Comment:
Load these modules directly, rather than doing so dynamically using `require` -- it seems to be marginally more stable when running tests, and since there isn't a memory gain for users from the dynamic method (everything is bundled into the kbase-narrative-min.js file), go for the more stable version.

Reply:
Yeah, seems like a good idea. Those are included in the `buildTools/loadAppWidgets`, right?

Reply:
yes, they are